glide.yaml

```diff
@@ -0,0 +1,26 @@
+package: github.com/openshift/must-gather
+import:
+- package: github.com/spf13/cobra
+  repo:    git@github.com:spf13/cobra
+  version: 7547e83b2d85fd1893c7d76916f67689d761fecb
+- package: github.com/spf13/pflag
+  repo:    git@github.com:spf13/pflag
+  version: 24fa6976df40757dce6aea913e7b81ade90530e1
+- package: k8s.io/client-go
+  repo:    https://github.com/kubernetes/client-go
+  version: cfd1e5200459ebe0b5d1c79193850b498fbb6135
+- package: k8s.io/cli-runtime
+  repo:    https://github.com/kubernetes/cli-runtime
+  version: 5702a69c455d3a129efbc2e86f8de61eb1af25a5
+- package: k8s.io/api
+  repo:    https://github.com/kubernetes/api
+  version: 173ce66c1e39d1d0f56e0b3347ff2988068aecd0
+- package: k8s.io/apimachinery
+  repo:    https://github.com/kubernetes/apimachinery
+  version: 001e837a013905514689c0e17c7065db5638974e
+- package: k8s.io/openshift/api
+  repo:    https://github.com/openshift/api
+  version: c3ef7090a436a4ce1334e8a0401083b072dc7ab8
+- package: k8s.io/xeonx/timeago
```

Comment:
This is weird, where did you get it from? Why the repo is different than package?

Reply:
oops, typo on my part. We use this in the utils/events.go file to format event times